lookbook/docs/patterns/02-forms.md.erb

```diff
@@ -44,6 +44,7 @@ input with the `:input_width` parameter.
 The options are:
 
 - `input_width: :auto` => `width: auto`
+- `input_width: :xsmall` => `max-width: min(144px, 100vw - 2rem)`
```

Comment:
input_width: :xlarge => max-width: min(640px, 100vw - 2rem) shouldn't be 
input_width: :xlarge => max-width: min(680px, 100vw - 2rem)?